components/about/contactForm.js

```diff
@@ -94,7 +116,7 @@ const ContactForm = ({ contactFormSubmitted }) => {
           onSubmit={handleSubmit(submit)}
         >
           {formError && (
-            <ErrorMessage msg='There was an error submitting the form. Please try again!' />
+            <ErrorMessage msg={`${formError}`} />
```

Comment:
is there some reacty reason `msg={formError}` wouldn't work ?

Reply:
done, u right